use-cases/manual-org-three-way.md

```diff
@@ -1,82 +1,350 @@
-# Secure for Cloud - Organizational Cloudtrail+SNS with S3 in a different account
+# Multi-AWS Accounts with Organizational CloudTrail and SNS with S3
```

Comment:
don't like this title much, but previous one neither 😓 

what about.... "Organizational setup -  Cloudtrail+SNS in organizational account and S3 in logs account"

Reply:
better